setup.py

```diff
@@ -145,6 +145,46 @@ def with_project_on_sys_path(self, func, func_args, func_kwargs):
         ''' override behavior, since we don't need to build '''
         return func(*func_args, **func_kwargs)
 
+class OpenShiftAnsibleGenerateValidation(Command):
+    ''' Command to run generated module validation'''
+    description = "Run generated module validation"
+    user_options = []
+        #('verify=', 'verify generated code'),
+    #]
+
+    def initialize_options(self):
+        ''' initialize_options '''
+        # Reason: Defining these attributes as a part of initialize_options is
+        # consistent with upstream usage
+        # Status: permanently disabled
+        # pylint: disable=attribute-defined-outside-init
+        #self.verify = True
+        pass
+
+    def finalize_options(self):
+        ''' finalize_options '''
+        # Reason: These attributes are defined in initialize_options and this
+        # usage is consistant with upstream usage
+        # Status: permanently disabled
+        # pylint: disable=attribute-defined-outside-init
+        pass
+
+    def run(self):
+        ''' run command '''
+        # find the files that call generate
+        generate_files = find_files('roles',
+                                    ['inventory',
+                                     'test',
+                                     'playbooks',
+                                     'utils'],
+                                    None,
+                                    'generate.py')
+
+        # call them with --verify
+        for gen in generate_files:
+            print('Checking generated module code: {0}'.format(gen))
+            subprocess.call([gen, '--verify'])
```

Comment:
Would it make sense to capture the output/error code here and aggregate it, instead of failing on the first error?